wfsim/strax_interface.py

```diff
@@ -593,8 +593,7 @@ def get_instructions(self):
             # Why epix removes many of the g4ids?
             # Remove nveto event if no tpc event of the same g4id is found
             if len(self.g4id) > 0:
-                nv_inst_to_keep = np.isin(self.instructions_nveto['g4id'], self.g4id[0])
-                nv_inst_to_keep &= (self.instructions_nveto['_last'] - self.instructions_nveto['_first']) > 0
+                nv_inst_to_keep = (self.instructions_nveto['_last'] - self.instructions_nveto['_first']) >= 0
```

Comment:
I think the cuts placed here should be the same as for the TPC events, using `entry_start` and `entry_stop`.

Reply:
What's the difference? I agree to use the same way among all detectors, but they should be exactly same, isn't it?